apps/desktop/src/renderer/helpers/formatting.ts

```diff
@@ -0,0 +1,51 @@
+import moment from 'moment';
+
+import { idStringMap } from 'f1-laps-js-bridge';
+
+export function formatValue(key: string, value: boolean | string | number) {
+    if (typeof value === 'boolean') {
```

Comment:
switch might be neater (not to mention typeof isn't very performant, so better to do it only once) here: 
```
const valueType = typeof value;

switch(valueType)....
```